src/components/renderers.js

```diff
@@ -11,7 +11,7 @@ export const DefaultItemRenderer = props => {
   const {item, ...rest} = props;
 
   return (
-    <span {...rest}>
+    <span {...rest} title={item.tooltip ? item.tooltip : ""}>
```

Comment:
I added it to have a tooltip if needed on the segments. For ex. if you have
![2021-04-14 08_18_49-demo js - Untitled (Workspace) - Visual Studio Code](https://user-images.githubusercontent.com/11580117/114658305-434fea80-9cfa-11eb-82ec-14fd98b819ae.png)

it will show
![2021-04-14 08_19_25-React Timeline 9000](https://user-images.githubusercontent.com/11580117/114658338-4c40bc00-9cfa-11eb-9c8b-4f83739e4985.png)

Reply:
Cool1 - didnt know that was supported by the title attr :)